packages/ag-charts-community/src/chart/interaction/zoomManager.ts

```diff
@@ -144,6 +148,18 @@ export class ZoomManager extends BaseManager<ZoomEvents['type'], ZoomEvents> imp
         this.applyChanges(callerId);
     }
 
+    panToBBox(callerId: string, seriesRect: BBox, target: BBoxValues) {
```

Comment:
nit: to match the other function styles in this file, helps make the intention of the api clearer

```suggestion
    public panToBBox(callerId: string, seriesRect: BBox, target: BBoxValues) {
```